draft-ietf-wimse-s2s-protocol.md

```diff
@@ -616,8 +616,8 @@ In order to reduce the risk of theft and replay the PoP should have a limited sc
 
 * Binding to a Timestamp or Nonce
```

Comment:
It looks like you removed the Nonce part.  While I'm a little sad to see this go we have not defined a mechanism to use it.  Does we want to still mention nonce?  if not we should remove it from the title.